codelens-sample/README.md

```diff
@@ -0,0 +1,18 @@
+# CodeLens Sample
+
+This is a sample extension that shows the usage of the CodeLens API.
+
+It is not intended as a production quality extension.
+
+- Create a new file
+- Write anything
+- Click on the CodeLens for action example
+- Can be enabled or disabled by command palete
+
+# How it works, what it shows?
+
+- The extension uses the [`CodeLens`](https://code.visualstudio.com/docs/extensionAPI/vscode-api#CodeLens)
+
+# How to run locally
+
+* open this folder in VS Code and press `F5`
```

Comment:
Please model it after the helloworld-sample by:

- Add a gif that demonstrates the usage
- Adding a "VS Code API" section
- Adding a "Running the Sample" section

https://github.com/microsoft/vscode-extension-samples/tree/master/helloworld-sample#vs-code-api